app/helpers/application_helper.rb

```diff
@@ -21,4 +21,28 @@ def related_media_card index, media, &block
       yield
     end
   end
+
+  def resource_users resource
+    user_ids = [0]
```

Comment:
there's a way to make this shorter and avoid repetition, e.g. I think this might work:
```
user_ids = (resource.publications + resource.activities).map do |item|
  item.users.pluck(:id) # or maybe even item.users_ids would work
end.flatten.uniq
```

Generally on collections often you can safely call `map` or `each` without testing for presence, because if they're empty then the block just won't execute and no exception is thrown.